### ISSUE TITLE
Check if the env file exists and if so append the variables if not create it

### ISSUE THREAD

**Issue by codelitt** [codelittinc/slacked#3]
Make sure we're not overwriting an existing .env file.

**Comment by codelitt** [comment]
@kaiomagalhaes this should be closed no?

**Comment by kaiomagalhaes** [comment]
@codelitt no because we are allowing the user to generate this file, but it will override the current one

**Comment by codelitt** [comment]
Okay. 

:fry_meme: not sure if bug or enhancement.

**Comment by kaiomagalhaes** [comment]
Makes sense to be a bug... I will change it

**Comment by thalessr** [comment]
In case the .env does not exist the generator creates a new one.
And in case the .env already exist and  does not contain any line starting with "SLACK_"  the generator appends the template into the .env.
I have test it locally and it worked.
I hope this solve the problem.  
```
module Slacked
  module Generators
    class InstallGenerator < ::Rails::Generators::Base
      source_root File.expand_path('../templates', __FILE__)

      desc 'Copy a Slacked file to your appplication'

      def create_configuration
        if !File.exist?('.env')
          copy_file('.env', '.env')
        elsif !File.readlines('.env').grep(/SLACK_/).any?
          template_dir = self.class.source_root
          lines = ''
          File.readlines("#{template_dir}/.env").each do |line|
            lines << line
          end
          append_file('.env', lines)
        end
      end
    end
  end
end
```

**Comment by kaiomagalhaes** [comment]
Closed by #14
